Jasmine/Common/ViewModels/LevelSelectViewModelProtocol.swift

```diff
@@ -0,0 +1,31 @@
+protocol LevelSelectViewModelProtocol {
+    /// The defaults levels in the game.
+    var defaultLevels: [GameInfo] { get }
+    /// The custom levels in the game.
+    var customLevels: [GameInfo] { get }
```

Comment:
I do need to listen if these information are changed (e.g. CRUD, high score changed, etc)